Solutions/MailGuard 365/Data Connectors/MailGuard365.json

```diff
@@ -0,0 +1,108 @@
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "MailGuard365_Threats_CL",
+            "baseQuery": "MailGuard365_Threats_CL"
+        }
+    ],
+    "sampleQueries": [
+        {
+            "description" : "All phishing threats stopped by MailGuard 365",
+            "query": "MailGuard365_Threats_CL \n | where Category == \"Phishing\""
+        },
+        {
+            "description" : "All threats summarized by sender email address",
+            "query": "MailGuard365_Threats_CL \n | summarize count() by Sender_Email_s"
+        },
+        {
+            "description" : "All threats summarized by category",
+            "query": "MailGuard365_Threats_CL \n | summarize count() by Category"
+        }
+    ],
+    "dataTypes": [
+        {
+            "name": "MailGuard365_Threats_CL",
+            "lastDataReceivedQuery": "MailGuard365_Threats_CL\n            | summarize Time = max(TimeGenerated)\n            | where isnotempty(Time)"
+        }
+    ],
+    "connectivityCriterias": [
+        {
+            "type": "IsConnectedQuery",
+            "value": [
+                "MailGuard365_Threats_CL\n            | summarize LastLogReceived = max(TimeGenerated)\n            | project IsConnected = LastLogReceived > ago(30d)"
+            ]
+        }
+    ],
+    "availability": {
+        "status": 1,
+        "isPreview": false
+    },
+    "permissions": {
+        "resourceProvider": [
+            {
+                "provider": "Microsoft.OperationalInsights/workspaces",
+                "permissionsDisplayText": "read and write permissions are required.",
+                "providerDisplayName": "Workspace",
+                "scope": "Workspace",
+                "requiredPermissions": {
+                    "write": true,
+                    "read": true,
+                    "delete": true
+                }
+            },
+            {
+                "provider": "Microsoft.OperationalInsights/workspaces/sharedKeys",
+                "permissionsDisplayText": "read permissions to shared keys for the workspace are required. [See the documentation to learn more about workspace keys](https://docs.microsoft.com/azure/azure-monitor/platform/agent-windows#obtain-workspace-id-and-key).",
+                "providerDisplayName": "Keys",
+                "scope": "Workspace",
+                "requiredPermissions": {
+                    "action": true
+                }
+            }
+        ]
+    },
+    "instructionSteps": [
+        {
+            "title": "Configure and connect MailGuard 365",
+            "description": "1. In the MailGuard 365 Console, click **Settings** on the navigation bar.\n2. Click the **Integrations** tab.\n3. Click the **Enable Microsoft Sentinel**.\n  - Enter your workspace id and primary key from the fields below, click **Finish**.\n5. For additional instructions, please contact MailGuard 365 support.",
```

Comment:
Hi @prathikc, the description needs to be fixed,  there is a small problem with the numbering, please take a look at image below.
![image](https://github.com/Azure/Azure-Sentinel/assets/120547590/e856040c-d88a-4c75-b7c3-f832677e5ad3)

Please repackage the solution post fixing.

Reply:
Hi @v-rbajaj , that was a good catch! I have made the changes and re-packaged the solution.